src/main/resources/sample/helloworld-settings.yml

```diff
@@ -1,4 +1,5 @@
 extensionName: hello-world
+shortExtensionName: hw
```

Comment:
Is `shortExtensionName` being used? I know we had talked about a separate values for extension unique id and an abbreviation for an extension, but is the extension unique id now also being treated as an abbreviation that would be prefixed to all action names? i.e. The route `_extensions/_hw/greet` is assigned the permissionable action name `hw:greet` and `hw` is the extension unique id

Reply:
Ok, I see now that extension name !== extension unique id. That sounds fine to me.

Reply:
removed the reference.